<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sys_info_opengl_widget</class>
 <widget class="QWidget" name="sys_info_opengl_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenGL information</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>OpenGL</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Vendor</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="vendorLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string notr="true"/>
          </property>
          <property name="statusTip">
           <string notr="true"/>
          </property>
          <property name="whatsThis">
           <string notr="true"/>
          </property>
          <property name="accessibleName">
           <string notr="true"/>
          </property>
          <property name="accessibleDescription">
           <string notr="true"/>
          </property>
          <property name="text">
           <string notr="true">Mesa3D OpenGL library</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Renderer</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="rendererLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string notr="true"/>
          </property>
          <property name="statusTip">
           <string notr="true"/>
          </property>
          <property name="whatsThis">
           <string notr="true"/>
          </property>
          <property name="accessibleName">
           <string notr="true"/>
          </property>
          <property name="accessibleDescription">
           <string notr="true"/>
          </property>
          <property name="text">
           <string notr="true">Mesa3D OpenGL Software Renderer</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Version</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="versionLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string notr="true"/>
          </property>
          <property name="statusTip">
           <string notr="true"/>
          </property>
          <property name="whatsThis">
           <string notr="true"/>
          </property>
          <property name="accessibleName">
           <string notr="true"/>
          </property>
          <property name="accessibleDescription">
           <string notr="true"/>
          </property>
          <property name="text">
           <string notr="true">6.66</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Extensions</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="extensionsBox">
        <property name="toolTip">
         <string notr="true"/>
        </property>
        <property name="statusTip">
         <string notr="true"/>
        </property>
        <property name="whatsThis">
         <string notr="true"/>
        </property>
        <property name="accessibleName">
         <string notr="true"/>
        </property>
        <property name="accessibleDescription">
         <string notr="true"/>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="undoRedoEnabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string notr="true">GL_EXT_SUPERCOOL1
GL_EXT_SUPERCOOL2</string>
        </property>
        <property name="backgroundVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
